src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,108 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTORY = 0.75f;
+    private int initialCapacity = 16;
```

Comment:
Any magic numbers should be constants. Your code should be easy to read. Move all hardcoded values to constant fields and give them informative names. For example: constants can be `defaultCapacity`, `loadFactor`...